build/BundledSdks.props

```diff
@@ -6,6 +6,7 @@
     <BundledSdk Include="Microsoft.NET.Sdk.Web" Version="$(MicrosoftNETSdkWebPackageVersion)" />
     <BundledSdk Include="Microsoft.NET.Sdk.Publish" Version="$(MicrosoftNETSdkPublishPackageVersion)" />
     <BundledSdk Include="Microsoft.NET.Sdk.Web.ProjectSystem" Version="$(MicrosoftNETSdkWebProjectSystemPackageVersion)" />
+    <BundledSdk Include="Microsoft.NET.Sdk.WPF" Version="$(MicrosoftNETSdkWPFPackageVersion)" />
```

Comment:
So, we are going with upper case WPF after all?

Reply:
No, that's a mistake (or maybe a place I forgot to update when switching to PascalCase)